homeassistant/components/wiz/light.py

```diff
@@ -79,14 +76,15 @@ def __init__(self, wiz_data: WizData, name: str) -> None:
         super().__init__(wiz_data, name)
         bulb_type: BulbType = self._device.bulbtype
         features: Features = bulb_type.features
-        color_modes = set()
+        self._attr_supported_color_modes: set = set()
```

Comment:
Should the set type be a bit more granular here?

Reply:
I adjusted the type hint.